app/src/main/java/com/google/android/stardroid/activities/dialogs/TimeTravelDialog.java

```diff
@@ -76,13 +78,17 @@ protected void onCreate(Bundle savedInstanceState) {
     Button changeDateButton = (Button) findViewById(R.id.pickDate);
     changeDateButton.setOnClickListener(new View.OnClickListener() {
         public void onClick(View v) {
+          if (SystemClock.elapsedRealtime() - LastClickTime < 1000) return;
+          LastClickTime = SystemClock.elapsedRealtime();
```

Comment:
Someone else has come up with a similar solution.  I'm not sure it's the way to go - what's special about 1000ms?

Reply:
Actually 1000 ms act like limit on simultaneous clicks. It differentiates two clicks if and only if their time difference is less than 1000.

Reply:
Yes, I can see that. I'm just wondering how the number was picked.